src/Native/Utils.js

```diff
@@ -357,6 +357,10 @@ Elm.Native.Utils.make = function(localRuntime) {
 
 	var toString = function(v)
 	{
+		if (v === null)
+			{
+				return '<internal structure>';
+			}
```

Comment:
The block above seems to have wrong indentation. Tabs vs spaces?

In addition, should this code also handle `undefined`? E.g. [with `== null`][1].

[1]: http://marijnhaverbeke.nl/blog/null-and-undefined.html